### PR TITLE
Issue-17562 default DNS cluster addon service's name fix

### PR DESCRIPTION
**Pull request by gireeshp**
This is a fix to Issue #17562

### CODE REVIEW

File: content/en/docs/concepts/services-networking/connect-applications-service.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ kube-dns   ClusterIP   10.0.0.10    <none>        53/UDP,53/TCP   8m
 
 If it isn't running, you can [enable it](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/kube-dns/README.md#how-do-i-configure-it).
 The rest of this section will assume you have a Service with a long lived IP
-(my-nginx), and a DNS server that has assigned a name to that IP (the CoreDNS
+(my-nginx), and a DNS server that has assigned a name to that IP (the Kube-dns
```
Comment (lojies) [inline content/en/docs/concepts/services-networking/connect-applications-service.md]: In fact, this cluster addon is CoreDNS with service name still kube-dns.

Comment (gireeshp) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Well, in that case, I think this part of the documentation is very confusing.
Screen Shot 2019-11-14 at 1 00 54 PM

A short description about it is named as "kube-dns", but it is CoreDNS behind the scene can be added (till the time the service name is corrected).
Also the link on "enable it" is pointing towards kube-dns deployment only (not CoreDNS). So if someone doesn't have it enabled, they will end up enabling kube-dns instead.

Comment (sftim) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Maybe “the CoreDNS cluster add-on (application name `kube-dns`)”?

The guide to enabling kube-dns should be about making sure that CoreDNS is running.

Comment (gireeshp) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Kube-dns cluster add-on has a README in https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns/kube-dns

But CoreDNS cluster add-on doesn't have one in https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns/coredns

Is there any good link about how to enable CoreDNS on Kubernetes?

**Comment by irvifa** [comment]
Hello @lojies any update on this?

File: content/en/docs/concepts/services-networking/connect-applications-service.md (hunk 1 above)
Comment (lojies) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Are these links help?
[https://github.com/coredns/deployment/tree/master/kubernetes](https://github.com/coredns/deployment/tree/master/kubernetes)
[https://kubernetes.io/docs/tasks/administer-cluster/coredns/](https://kubernetes.io/docs/tasks/administer-cluster/coredns/)

Comment (gireeshp) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Yes. These links look good. Let's give the user both. Updating the PR..

**Comment by gireeshp** [comment]
@lojies & @sftim updated the PR with suggested changes. Please see if it looks good.

**Comment by tengqm** [comment]
@sftim pls leave a lgtm if you are okay with the change.

File: content/en/docs/concepts/services-networking/connect-applications-service.md [hunk 2]
```diff
@@ -201,11 +201,8 @@ NAME       TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)         AGE
 kube-dns   ClusterIP   10.0.0.10    <none>        53/UDP,53/TCP   8m
 ```
 
-If it isn't running, you can [enable it](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/kube-dns/README.md#how-do-i-configure-it).
 The rest of this section will assume you have a Service with a long lived IP
-(my-nginx), and a DNS server that has assigned a name to that IP (the CoreDNS
-cluster addon), so you can talk to the Service from any pod in your cluster using
-standard methods (e.g. gethostbyname). Let's run another curl application to test this:
+(my-nginx), and a DNS server that has assigned a name to that IP. Here we use the CoreDNS cluster addon (application name ```kube-dns```), so you can talk to the Service from any pod in your cluster using standard methods (e.g. gethostbyname). If it isn't running, you can enable it referring to [this](https://github.com/coredns/deployment/tree/master/kubernetes) or [this](https://kubernetes.io/docs/tasks/administer-cluster/coredns/) links. Let's run another curl application to test this:
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/connect-applications-service.md]: This is OK technically and although it doesn't follow the documentation style guide, it _is_ mostly in line with the way the rest of the page is written.

@gireeshp if you want to you can accept this suggestion:
```suggestion
(my-nginx), and a DNS server that has assigned a name to that IP. Here we use the CoreDNS cluster addon (application name `kube-dns`), so you can talk to the Service from any pod in your cluster using standard methods (e.g. `gethostbyname()`). If CoreDNS isn't running, you can enable it referring to the [CoreDNS README](https://github.com/coredns/deployment/tree/master/kubernetes) or [Installing CoreDNS](/docs/tasks/administer-cluster/coredns/#installing-coredns). Let's run another curl application to test this:
```

If you prefer the change is, let me know and I'll add /lgtm

Comment (gireeshp) [reply content/en/docs/concepts/services-networking/connect-applications-service.md]: Agree with your suggested update @sftim. Committed it.

**Comment by sftim** [comment]
This seems better than the current text
/lgtm

**Pull request merged**